ui/src/utils/Timestamp.js

```diff
@@ -362,7 +362,7 @@ export function getDayOfYear (timestamp) {
 export function getWorkWeek (timestamp) {
   let date
   if (timestamp.year === 0) {
-    date = Date.UTC(new Date())
+    date = NaN
```

Comment:
The intent of this is if `timestamp.year` is 0 (default after Timestamp initialization), then to use the current date. This could break some things where the intent is to use the current date.